Assets/02_Student Folders/TayfunAygun_Assets/Prefabs/Projectiles.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Oh yeah why did you change/add stuff in somebody else his folder?

Reply:
This should not have happened, will fix

Reply:
I fixed the merge so that i will take what is in the master branch and not whatever i did in my branch.